lua/hbac/init.lua

```diff
@@ -92,7 +92,38 @@ local function close_unused()
 	end
 end
 
+
+-- Create commands to add and remove buffers from the persistant_buffers table
```

Comment:
Are you using some kind of AI assisted development? 😆 Otherwise the comment is superfluous 
```suggestion

```